File: NodeJS/interactions/fill_form.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com/');
  await page.fill('input#name', 'John');
  await page.fill('input#surname', 'Doe');
  await page.click('button[type=submit]');
  console.log(await page.content());
  await browser.close();
})();

File: Python/interactions/fill_form.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://example.com/")
    page.fill("input#name", "John")
    page.fill("input#surname", "Doe")
    page.click("button[type=submit]")
    print(page.content())
    browser.close()
